packages/primitives/src/Button/Button.tsx

```diff
@@ -0,0 +1,96 @@
+import styled from '@emotion/styled'
+import { SerializedStyles } from '@emotion/serialize'
+import {
+  space,
+  SpaceProps,
+  color,
+  ColorProps,
+  typography,
+  TypographyProps,
+  border,
+  BorderProps,
+  position,
+  PositionProps,
+  shadow,
+  ShadowProps,
+} from 'styled-system'
+import { createShouldForwardProp } from '@styled-system/should-forward-prop'
+import { variant, sx } from '../Box/Box'
+import { css, Theme } from 'theme-ui'
+
+export const shouldForwardProp = createShouldForwardProp([
+  ...(space.propNames as string[]),
+  ...(color.propNames as string[]),
+  ...(typography.propNames as string[]),
+  ...(border.propNames as string[]),
+  ...(position.propNames as string[]),
+  ...(shadow.propNames as string[]),
+])
+
+/**
+ * base
+ *
+ * Generate the base CSS for the button component
+ * that is aware of the Theme UI theme
+ *
+ * @param props
+ */
+export function base({ theme }: { theme: Theme }): SerializedStyles {
+  return css({
+    px: 3,
+    py: 2,
+    color: 'white',
+    bg: 'primary',
+    border: 0,
+    borderRadius: 4,
```

Comment:
Should we add the default radius to the theme and make this `borderRadius: 'default'`

`radii: {
    default: 4,
    circle: 99999,
  },`

Reply:
Yep that’s a good call!